src/calendar/day/index.tsx

```diff
@@ -68,8 +78,8 @@ export default class Day extends Component {
   }
 
   getAccessibilityLabel = memoize((day, marking, isToday) => {
-    const today = XDate.locales[XDate.defaultLocale].today;
-    const formatAccessibilityLabel = XDate.locales[XDate.defaultLocale].formatAccessibilityLabel;
+    const today = _.get(XDate, 'locales.[XDate.defaultLocale].today');
+    const formatAccessibilityLabel = _.get(XDate, 'locales[XDate.defaultLocale].formatAccessibilityLabel');
```

Comment:
Did you test these changes? I see that the first one includes `locales**.**[XDate.defaultLocale]` and the other one `locales[XDate.defaultLocale]`...

Reply:
fixed :)